### ISSUE TITLE
Need to update for Safari 12 / macOS 10.14 Mojave Support

### ISSUE THREAD

**Issue by KZeni** [arnoappenzeller/PiPifier#60]
It appears the old safari extension setup no longer works in Safari 12 or newer (which ships with macOS 10.14 or newer & can be used on older versions). The extension appears to simply not be offered as an extension in Safari at this point. As such https://developer.apple.com/documentation/safariservices/safari_app_extensions/converting_a_safari_extension_to_a_safari_app_extension?language=objc should be followed to make this extension working again.

**Comment by arnoappenzeller** [comment]
PiPifier already is a modern Safari App extension. 
I didn’t have the time to test it on Mojave but I’ll try to do it before release.

**Comment by KZeni** [comment]
Ah, I suppose mis-diagnosed why it wasn't being offered in the extensions listing. Sorry about that. I've updated the issue's title appropriately.

**Comment by KZeni** [comment]
Okay... I just noticed Safari disabled the Develop menu bar item (previously set to show), and I went into the settings to re-enable it & I then saw the extension is showing in in the list again. I didn't restart my computer or manually apply any updates to Safari (did close & reopen Safari at one point, though) so I'm a bit surprised by this. As such, I'll close this issue since it actually seems to be working properly now (might've been a post-update cache issue?)

As an aside, 1Blocker was experiencing the same thing, but it appears their extension is showing in the extension list now as well.
